tests/system/providers/google/leveldb/example_leveldb.py

```diff
@@ -23,8 +23,16 @@
 import os
 from datetime import datetime
 
+import pytest
+
 from airflow import models
-from airflow.providers.google.leveldb.operators.leveldb import LevelDBOperator
+from airflow.exceptions import AirflowOptionalProviderFeatureException
+
+try:
+    from airflow.providers.google.leveldb.operators.leveldb import LevelDBOperator
+except AirflowOptionalProviderFeatureException:
+    pytest.skip("LevelDB not available", allow_module_level=True)
```

Comment:
Actually we could install selected packages into ARM CI image `libleveldb1d`, `libleveldb-dev` and after that we might successfully install `plyvel`. At least it work when I migrate this test to pytest.

```console
root@e3f4bad5e095:/opt/airflow# apt-get install libleveldb1d libleveldb-dev -y
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  libsnappy1v5
Suggested packages:
  leveldb-doc
The following NEW packages will be installed:
  libleveldb-dev libleveldb1d libsnappy1v5
0 upgraded, 3 newly installed, 0 to remove and 5 not upgraded.
Need to get 341 kB of archives.
After this operation, 1521 kB of additional disk space will be used.
Get:1 http://deb.debian.org/debian bullseye/main arm64 libsnappy1v5 arm64 1.1.8-1 [17.2 kB]
Get:2 http://deb.debian.org/debian bullseye/main arm64 libleveldb1d arm64 1.22-3 [132 kB]
Get:3 http://deb.debian.org/debian bullseye/main arm64 libleveldb-dev arm64 1.22-3 [192 kB]
Fetched 341 kB in 0s (1114 kB/s)       
Selecting previously unselected package libsnappy1v5:arm64.
(Reading database ... 30889 files and directories currently installed.)
Preparing to unpack .../libsnappy1v5_1.1.8-1_arm64.deb ...
Unpacking libsnappy1v5:arm64 (1.1.8-1) ...
Selecting previously unselected package libleveldb1d:arm64.
Preparing to unpack .../libleveldb1d_1.22-3_arm64.deb ...
Unpacking libleveldb1d:arm64 (1.22-3) ...
Selecting previously unselected package libleveldb-dev:arm64.
Preparing to unpack .../libleveldb-dev_1.22-3_arm64.deb ...
Unpacking libleveldb-dev:arm64 (1.22-3) ...
Setting up libsnappy1v5:arm64 (1.1.8-1) ...
Setting up libleveldb1d:arm64 (1.22-3) ...
Setting up libleveldb-dev:arm64 (1.22-3) ...
Processing triggers for libc-bin (2.31-13+deb11u5) ...
root@e3f4bad5e095:/opt/airflow# pip install plyvel
Collecting plyvel
  Downloading plyvel-1.5.0.tar.gz (152 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 152.3/152.3 kB 1.7 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
Building wheels for collected packages: plyvel
  Building wheel for plyvel (setup.py) ... done
  Created wheel for plyvel: filename=plyvel-1.5.0-cp37-cp37m-linux_aarch64.whl size=92562 sha256=9ff2a2511ddd6208c681b4f1da198fe6594d57648b5cd87d7d3019c45c22ffc9
  Stored in directory: /tmp/pip-ephem-wheel-cache-57dgr3zr/wheels/d7/4e/63/7cd883b7e898b245fd60b5db4fbe805fe0c197e24024aa8c97
Successfully built plyvel
Installing collected packages: plyvel
Successfully installed plyvel-1.5.0
```

However we have in `setup.py` strict limitation about leveldb extra:
https://github.com/apache/airflow/blob/8e0df8881f22dd5c4c0ea71e7a9cd35b32889f47/setup.py#L307

Which is inconsistent with provider extra:
https://github.com/apache/airflow/blob/8e0df8881f22dd5c4c0ea71e7a9cd35b32889f47/airflow/providers/google/provider.yaml#L1065-L1067

Reply:
Yeah we can attempt to do it as next step. The difference is that the first one was needed for image building that's why aarch exclusion - the second is just extra specification for google provider (and those two might not be synchronized indeed). But if we can build it in the way you described, that would be better than excluding it. I will try it in a moment.